playbooks/update-nginx-admin-password.yml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: role=control
+  serial: 1
+  tasks:
+    - name: encrypt admin password
+      sudo: yes
+      run_once: yes
+      shell: htpasswd -Bnb admin {{ nginx_admin_password | quote }} | cut -f 2 -d ':'
+      register: nginx_admin_password_encrypted
```

Comment:
Could we use two different variable names, instead of reassigning `nginx_admin_password_encrypted`?